src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -489,6 +506,72 @@ private function addServiceMethodCalls($id, Definition $definition, $variableNam
         return $calls;
     }
 
+    private function addServiceOverriddenGetters($id, Definition $definition)
+    {
+        if (!isset($this->classResources[$class = $definition->getClass()])) {
+            $this->classResources[$class] = new \ReflectionClass($class);
```

Comment:
the property name is weird here. ``$this->classResources`` does not contain resource objects.